src/core/commands/cloud_output.py

```diff
@@ -4,28 +4,26 @@
 
 from core.utils.logger import log
 
+
 def cloud_output_command_cli(args):
     config = args[0]
+    # ANIBAL missing arguments here
```

Comment:
We need a solution for the following problem:
The ruuid is of the form <>::<>::<> where each `<>` represents a namespace. Generally the form will be <top-provider>::<sub-namespace>::<descriptive resource name> like cdev::simple::function. This is structure breaks up the CLI flow of typing <component>.<ruuid>.<cdev_name>.<output_key> because the <ruuid> is such a different structure. 

As a stop gap, I just assume all ruuid start with `cdev::simple::` , but this is obviously not a long term solution. Some things I have thought of before were just keeping it as `<component>.<cdev_name>.<output_key>` then a separate flag `-ruuid or -R` where you enter the ruuid. This feels less ergonomic though.

Reply:
See line 23/24 for where the hard coding of `cdev::simple::` is